public/scripts/controllers/AlbumsIndexController.js

```diff
@@ -0,0 +1,67 @@
+angular
+  .module('tunely')
+  .controller('AlbumsIndexController', AlbumsIndexController);
+
+AlbumsIndexController.$inject = ['$http'];
+
+function AlbumsIndexController ($http) {
+  var vm = this;
+  vm.newAlbum = {};
+  vm.newAlbum = {
+    name: 'Viva Hate',
+    artistName: 'Morrissey'
+  };
+
+  $http({
+    method: 'GET',
+    url: '/api/albums'
+  }).then(function successCallback(response) {
+    vm.albums = response.data;
+  }, function errorCallback(response) {
+    console.log('There was an error getting the data', response);
+  });
+
+  vm.createAlbum = function () {
+    $http({
+      method: 'POST',
+      url: '/api/albums',
+      data: vm.newAlbum,
+    }).then(function successCallback(response) {
+      vm.albums.push(response.data);
+    }, function errorCallback(response) {
+      console.log('There was an error posting the data', response);
+    });
+  }
+
+  vm.editAlbum = function (album) {
```

Comment:
I think either we decide to have a controller that handles all of a resource Or we have a controller per action.  But we need to decide.  

I believe 25 stuck with one controller per resource (like rails and now like our new node style)  - but the other pattern may be more common in Angular City.